g4f/Provider/bing/create_images.py

```diff
@@ -194,48 +178,47 @@ def get_cookies_from_browser(proxy: str = None) -> Dict[str, str]:
 
 class CreateImagesBing:
     """A class for creating images using Bing."""
-
-    _cookies: Dict[str, str] = {}
     
-    @classmethod
-    def create_completion(cls, prompt: str, cookies: Dict[str, str] = None, proxy: str = None) -> Generator[str, None, None]:
+    def __init__(self, cookies: dict[str, str] = {}, proxy: str = None) -> None:
+        self.cookies = cookies
+        self.proxy = proxy
+    
+    def create_completion(self, prompt: str) -> Generator[ImageResponse, None, None]:
         """
         Generator for creating imagecompletion based on a prompt.
 
         Args:
             prompt (str): Prompt to generate images.
-            cookies (Dict[str, str], optional): Cookies for the session. If None, cookies are retrieved automatically.
-            proxy (str, optional): Proxy configuration.
 
         Yields:
             Generator[str, None, None]: The final output as markdown formatted string with images.
         """
-        loop = get_event_loop()
-        cookies = cookies or cls._cookies or get_cookies(".bing.com")
+        cookies = self.cookies or get_cookies(".bing.com")
```

Comment:
Consider adding type hints to the `cookies` parameter of the `create_completion` method.